packages/solid/components/Keyboard/Keyboard.stories.tsx

```diff
@@ -41,625 +55,142 @@ export const Basic: Story = {
   },
+  parameters: {
+    docs: {
+      description: {
+        story: 'a simple keyboard for case-insensitive input'
+      }
+    }
+  },
   args: {
     states: 'focus',
     autofocus: true,
     centerKeyboard: false,
     centerKeys: false,
-    width: 1280,
-    formats: {
-      default: [
-        [
-          '1',
-          '2',
-          '3',
-          '4',
-          '5',
-          '6',
-          '7',
-          '8',
-          '9',
-          '0',
-          {
-            title: 'Delete',
-            size: 'md'
-          }
-        ],
-        ['q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p'],
-        ['a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l'],
-        ['z', 'x', 'c', 'v', 'b', 'n', 'm'],
-        [
-          {
-            title: 'Clear',
-            size: 'lg'
-          },
-          {
-            title: 'Space',
-            size: 'xl'
-          },
-          {
-            title: 'Done',
-            size: 'lg'
-          }
-        ]
-      ]
+    width: 1280
+  }
+};
+
+export const KeyboardEmail: Story = {
+  render: args => {
+    return <Email {...args} />;
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'a full featured keyboard with helpful shortcuts for email addresses'
+      }
     }
+  },
+  args: {
+    states: 'focus',
+    autofocus: true,
+    centerKeys: false,
+    centerKeyboard: false,
+    width: 1280
   }
 };
 
-export const KeyboardSimple1: Story = {
+export const KeyboardQwerty: Story = {
   render: args => {
-    return <KeyboardSimple {...args} />;
+    return <Qwerty {...args} />;
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'a full featured keyboard for more complex text input'
+      }
+    }
   },
   args: {
     states: 'focus',
+    autofocus: true,
     centerKeys: false,
     centerKeyboard: false,
+    width: 1280
+  }
+};
+
+export const KeyboardNumber: Story = {
+  render: args => {
+    return <Number {...args} />;
+  },
+  parameters: {
+    docs: {
+      description: {
+        story: 'a simple keyboard for numeric input'
+      }
+    }
+  },
+  args: {
+    states: 'focus',
     autofocus: true,
-    width: 1280,
-    formats: {
-      default: [
-        [
-          '1',
-          '2',
-          '3',
-          '4',
-          '5',
-          '6',
-          '7',
-          '8',
-          '9',
-          '0',
-          {
-            title: 'Delete',
-            size: 'md'
-          }
-        ],
-        ['q', 'w', 'e', 'r', 't', 'y', 'u', 'i', 'o', 'p'],
-        ['a', 's', 'd', 'f', 'g', 'h', 'j', 'k', 'l'],
-        ['z', 'x', 'c', 'v', 'b', 'n', 'm'],
-        [
-          {
-            title: 'Clear',
-            size: 'lg'
-          },
-          {
-            title: 'Space',
-            size: 'xl'
-          },
-          {
-            title: 'Done',
-            size: 'lg'
-          }
-        ]
-      ]
+    centerKeys: false,
+    centerKeyboard: false,
```

Comment:
It could be nice to set these centering props to true for all keyboard stories for look and feel, just a peanut gallery thought 🥜

Reply:
leaving these false for now - there's a bug causing the stories to animate in every time if they're true and I'd rather resolve that in another ticket before making center the default

Reply:
yup - great call!